guides/common/modules/proc_republishing-content-view-metadata.adoc

```diff
@@ -0,0 +1,10 @@
+[id="Republishing_Content_View_Metadata_{context}"]
+= Republishing Content View Metadata
+
+Use this procedure to republish Content View metadata.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content View* > *Versions* tab.
+. Select the Content View version metadata you want to republish, click the vertical ellipsis icon, and click *Republish repository metadata*.
```

Comment:
```suggestion
. In the {ProjectWebUI}, navigate to *Content* > *Content Views*.
. Select a Content View.
. On the *Versions* tab, select a Content View version.
. To republish metadata for the Content View version, click *Republish repository metadata* from the vertical ellipsis icon.
```

I've tried to test this on Foreman 3.7. I am unsure about the wordiing of the fourth step in my suggestion.